README.md

```diff
@@ -118,6 +123,7 @@ $ rails new . -d postgresql
 
 ```
 Your answer...
+rails new turnr creates a new project folder when the project is forked and clone while rails new . -d postgresql creates and saves a file with its own repository.
```

Comment:
`rails new tunr` creates a new directory and generates all the expected files.  When we fork and clone a project, we expect all of those files to be available in the working directory created during during the clone, therefore we do NOT use both `rails new` and `git clone`.  
The `-d postgresql` flag is used to configure the newly generated rails app to use the Postgres database, instead of the default db.